### DISCUSSION TITLE
Alternative hotkeys for scrolling up or down in the tree

### DISCUSSION THREAD

**Discussion by bytewife** [piroor/treestyletab#3226] [Ideas]
First off, thank you for the extension! Secondly, is it currently possible to map the tree-scrolling action to other keys (besides `Up Arrow` & `Down Arrow`)? I greatly prefer using `Tab` and `Shift+Tab` over the arrow keys for ergonomics. What are your thoughts?

**Comment by piroor** [comment]
There is a UI to re-assign shortcut keys for features of TST. You should click the gear icon in the addons manager and choose "Manage Extension Shortcuts".
![image](https://user-images.githubusercontent.com/70062/195931050-cd71c183-d651-4d48-81b3-6924af792229.png)
![image](https://user-images.githubusercontent.com/70062/195931075-94b773b7-20b7-43e2-b58b-614e9fc23b65.png)
But please note that Firefox doesn't allow us to assign shortcuts already in use by Firefox itself.
(See also https://bugzilla.mozilla.org/show_bug.cgi?id=1325692 )

**Comment by bytewife** [comment]
Thank you!
